easybuild/easyconfigs/a/Albacore/Albacore-2.0.2-intel-2017a-Python-3.6.1.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'Bundle'
+
+name = 'Albacore'
+version = '2.0.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://community.nanoporetech.com/protocols/albacore-offline-basecalli/v/abec_2003_v1_revz_29nov2016'
+description = """Albacore is a software project that provides an entry point to the Oxford Nanopore basecalling
+ algorithms."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+pyver = '3.6.1'
+pymajmin = ''.join(pyver.split('.')[:2])
+dependencies = [
+    ('Python', pyver),
+    ('numpy', '1.13.1', versionsuffix),
```

Comment:
that's part of the `Python`?

Reply:
The `numpy` version included there is too old, the `.whl` used here requires numpy 1.13.x